lib/simple_forms_api/form_remediation/configuration/vff_config.rb

```diff
@@ -7,7 +7,7 @@ module FormRemediation
     module Configuration
       class VffConfig < Base
         def s3_settings
-          Settings.vff_simple_forms.aws
```

Comment:
@pennja Will this setting be used anywhere now? If not, you can delete it from the manifest repo. I see you added it in https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3116/ and https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3117

Reply:
Oof, great catch. This change and the changes to the uploader were never supposed to be added. Those were local changes for testing purposes. I've reverted those changes out. Thank you!

Reply:
oh wow. I'm glad I said something!